From 69830b97d0d6d73eb59af8ff1d8b1f6fbb1caa60 Mon Sep 17 00:00:00 2001
From: tropf <tropf@noreply.codeberg.org>
Date: Tue, 8 Aug 2023 20:34:46 +0200
Subject: [PATCH 3/9] fix path search

Some common paths are searched which do no exist on NixOS. This throws
errors, hence this path search is fixed.
---
 inkstitch.py | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/inkstitch.py b/inkstitch.py
index a69caa0c..920dfc61 100644
--- a/inkstitch.py
+++ b/inkstitch.py
@@ -23,8 +23,9 @@ if getattr(sys, 'frozen', None) is None:
     else:
         extensions_path = "/usr/share/inkscape/extensions"
 
-    sys.path.remove(extensions_path)
-    sys.path.append(extensions_path)
+    if extensions_path in sys.path:
+        sys.path.remove(extensions_path)
+        sys.path.append(extensions_path)
 
 from inkex import errormsg
 from lxml.etree import XMLSyntaxError
-- 
2.40.1

